src/mlpack/methods/neat/gene_impl.hpp

```diff
@@ -48,6 +48,17 @@ void ConnectionGene::Mutate(const double mutationSize)
   weight += mutationSize * arma::randn<double>();
 }
 
+template <typename Archive>
+void ConnectionGene::serialize(Archive& ar,
+                                const unsigned int /* version */)
```

Comment:
Pedantic style issue number one: the second parameter should align with the first one.